firmware/examples/SdCard/source/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include <cstdint>
+
+#include "L3_HAL/sddriver.hpp"
+#include "utility/log.hpp"
+#include "utility/time.hpp"  // halt()
+#include "utility/debug.hpp"  // debug::Hexdump()
+
+const uint8_t kLongText[512] = R"(Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit es se cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat no n proident, sunt in culpa qui officia deserunt mollit anim id est laborum.)";
+const uint8_t kHelloWorld[512] = "Hello World!";
+
+int main(void)
+{
+    LOG_INFO("BEGIN SD Card Driver Example...");
+
+    Sd sdcard(Sd::DebugSdCard{});
+    Sd::CardInfo_t sdCardInfo;
+    uint8_t buffer[512];
+    sdcard.Initialize();
+    sdcard.Mount(&sdCardInfo);
```

Comment:
Since this returns a boolean value, you should check at this point if the sdcard mount is mounted properly, and if it is not, you should halt the system.
```suggestion
    SJ2_ASSERT_FATAL(sdcard.Mount(&sdCardInfo), "Failed to mount SD Card");
```

Reply:
Applying to my local staging branch.